skimage/transform/registration.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+from scipy.optimize import basinhopping, minimize
+
+from .pyramids import pyramid_reduce
+from ._warps import warp, SimilarityTransform
+
+__all__ = ['register', 'p_to_matrix', 'matrix_to_p']
+
+
+def _gaussian_pyramid(image, levels=6):
+    """
+    Creates a gaussian pyramid for an image
+    
+    Parameters
+    ----------
+    image : (M, N) ndarray
+        Input image with is used as a base for the pyramid.
+    levels : int, optional (default 6)
+        Int equals to the amount of times the image is blurred and reduced
+    
+    Returns
+    -------
+    pyramid: array
+        Ordered array of images of size level starting with the most blurred
+        
+    """
+    pyramid = levels*[None]
+    pyramid[-1] = image
+
+    for level in range(levels-2, -1, -1):
+        image = pyramid_reduce(image, sigma=2/3)
+        pyramid[level] = (image)
+
+    return pyramid
```

Comment:
You can use `reversed(pyramid_gaussian(image, levels=6))` instead of writing this function.